contrib/kube-prometheus/jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```diff
@@ -4,6 +4,22 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
   _config+:: {
     namespace: 'default',
 
+    kubeStateMetrics+:: {
+      // when this is an empty string, you get the default set
+      collectors: '',
+      scrapeTimeout: '',
```

Comment:
I just checked, the default scrape timeout is `10s`, I think we can have this default to the value of `scrapeInterval` for kube-state-metrics. And generally I think `1m` interval is probably a more universal choice. Even `2m` is still fine with the dashboards and alerting rules that we currently have (and more generally more than `2m` scrape interval is not advised).

Reply:
I'm not sure if you're suggesting I merge `scrapeTimeout` and `scrapeInterval` into a single variable, or if I should keep them split and set them to better initial values. (Happy to do either, and I don't have much preference)

Reply:
I think we should keep them split but have them default on the same value.